features/step_definitions/utils.rb

```diff
@@ -0,0 +1,29 @@
+
+Then (/^I load the form payload at "([^\"]*)" onto the phone as a "([^\"]*)" form$/) do |payload_path, form_status|
```

Comment:
I ended up not using this for this test because I'm doing it directly in a form that I can fill out instead, but I confirmed that this util method works and seems very likely to be useful in the future so I didn't want to delete it